runner/dl_checkpoints.sh

```diff
@@ -133,9 +133,9 @@ function build_tensorrt_models() {
 
   # Dreamshaper-8-Dmd-1kstep
   # TODO: Remove the script download with curl. It should already come in the base image once eliteprox/comfystream#1 is merged.
```

Comment:
`TODO:` got confusing as now we don't use eliteprox's comfyui. Now it's yondon's.

There's https://github.com/eliteprox/comfystream/blob/main/src/comfystream/scripts/build_trt.py on the `eliteprox`. I guess it doesn't do what we want?

Reply:
It looks it was confusing from the beginning https://github.com/livepeer/ai-runner/pull/405/files#diff-49045665a4e2315c6702f467b07ebeff183e1a7db386704f45a5486da05b1857R147

Reply:
Yeah I wasn't sure about my change to this version of `build_trt.py` @pschroedl could you check? I think we just needed a newer version with the `/workspace` prefix update.